tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -218,32 +216,29 @@ export class Ide {
         await this.waitStatusBarContains(expectedTextInStatusBar, 20000);
     }
 
-    async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+    async closeAllNotifications(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        const statusBarNotificationsLocator: By = By.xpath(`//div[@id="theia-statusBar"]/div[@class="area right"]//div[contains(@title, 'Notification')]`);
+        const notificationCenterActionCloseAllButtonLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center open')]//ul[@class="theia-notification-actions"]/li[@class="clear-all"]`);
+        const notificationCenterClosedLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center closed')]`);
+        const notificationCenterContainerLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center')]//div[@class='theia-notification-list']`);
         Logger.debug('Ide.closeAllNotifications');
 
-        if (! await this.driverHelper.isVisible(notificationLocator)) {
-            return;
-        }
-
-        const notifications: WebElement[] = await this.driverHelper.waitAllPresence(notificationLocator);
-        const notificationsCapacity: number = notifications.length;
-
-        for (let i = 1; i <= notificationsCapacity; i++) {
-            const notificationLocator: By = By.xpath('//div[@class=\'theia-Notification\']//button[text()=\'Close\']');
-
-            try {
-                await this.driverHelper.waitAndClick(notificationLocator);
-            } catch (err) {
-                if (err instanceof error.TimeoutError) {
-                    console.log(`The '${notificationLocator}' element is not visible and can't be clicked`);
-                    continue;
-                }
-
-                throw err;
+        Logger.trace(`Ide.closeAllNotifications driverHelper click Notification Center icon.`);
+        await this.driverHelper.waitAndClick(statusBarNotificationsLocator, 5_000);
+        Logger.trace(`Ide.closeAllNotifications driverHelper Notifications Center click closeAllNotifications.`);
+        await this.driverHelper.waitAndClick(notificationCenterActionCloseAllButtonLocator, 5_000);
+        Logger.trace(`Ide.closeAllNotifications driverHelper waitng for Notification Center to close.`);
+        await this.driverHelper.waitPresence(notificationCenterClosedLocator, 5_000);
+        Logger.trace(`Ide.closeAllNotifications driverHelper test if any notifications remain in Notification Center.`);
+        for (var attempts: number = 0; attempts < (timeout / 1_000); attempts++) {
```

Comment:
I think may be replaced by:
```
        await this.driverHelper.waitDisappearance(locator, timeout)
```
And locator should consist of 
notificationCenterContainerLocator + /{childTag}  

for me, it looks much simple than 
```
        for (var attempts: number = 0; attempts < (timeout / 1_000); attempts++) {
            if (await this.driverHelper.hasChildren(notificationCenterContainerLocator)) {
                Logger.trace(`Ide.closeAllNotifications notifications still present. Trying again in 1s.`);
                this.driverHelper.wait(1_000);
            } else {
                return;
            }
        }
        throw new error.TimeoutError(`Ide.closeAllNotifications there are still notifications present after closing. Timed out after ${timeout}ms`);
```

Reply:
it's not easy to append a path to an already constructed locator. the string it generates looks like `By(xpath, <locator>)`
It's better to have a separate one just for the notifications.